libs/barista-components/table/src/table.ts

```diff
@@ -321,4 +332,41 @@ export class DtTable<T> extends _DtTableBase<T> implements OnDestroy {
 
   /** CSS class added to any row or cell that has sticky positioning applied. */
   protected stickyCssClass = 'dt-table-sticky';
+
+  exportCSV(): void {
+    let ds = this.dataSource as DtTableDataSource<T>;
```

Comment:
Most of the variable declarations here could be `const. We try to use constants wherever possible